catalog/core/catalog-core-solrcommands/src/test/java/org/codice/ddf/commands/solr/TestBackupCommand.java

```diff
@@ -160,4 +507,66 @@ private HttpResponse prepareResponse(int statusCode, String responseBody) {
 
         return httpResponse;
     }
+
+    private static void createDefaultMiniSolrCloudCluster() throws Exception {
+        createMiniSolrCloudCluster();
+        uploadDefaultConfigset();
+        createDefaultCollection();
+    }
+
+    private static void createMiniSolrCloudCluster() throws Exception {
+        miniSolrCloud = new MiniSolrCloudCluster(1,
+                getBaseDirPath(),
+                JettyConfig.builder()
+                        .setContext("/solr")
+                        .build());
+        miniSolrCloud.getSolrClient().connect();
+    }
+
+    private static void uploadDefaultConfigset() throws InterruptedException, IOException, KeeperException {
```

Comment:
Would it make sense to change these methods to "throws Exception" since they throw multiple types.